src/DataViews/include/DataViews/DataView.h

```diff
@@ -58,12 +61,25 @@ constexpr std::string_view kMenuActionCopySelection = "Copy Selection";
 constexpr std::string_view kMenuActionExportToCsv = "Export to CSV";
 constexpr std::string_view kMenuActionExportEventsToCsv = "Export events to CSV";
 
+static constexpr const char* kFieldSeparator = ",";
+// CSV RFC requires lines to end with CRLF
+static constexpr const char* kLineSeparator = "\r\n";
+
 // Values in the DataView may contain commas, for example, functions with arguments. We quote all
 // values in the output and also escape quotes (with a second quote) in values to ensure the CSV
 // files can be imported correctly in spreadsheet applications. The formatting follows the
 // specification in https://tools.ietf.org/html/rfc4180.
 std::string FormatValueForCsv(std::string_view value);
 
+template <typename Range>
+ErrorMessageOr<void> WriteLineToCSV(const orbit_base::unique_fd& fd, const Range& cells) {
```

Comment:
This function should probably have a test.

Reply:
As promised, tests will be added in the next PR.